addons/nachschub/configs/CfgVehicles_USA.hpp

```diff
@@ -99,6 +129,7 @@ class TB_supply_usa_spezial : WRAPPER_NAME(Box_East_Wps_F)
         // 12.7
```

Comment:
Du hast beim Virratslager die Kisten von Spezial in Präzision umbenannt aber nicht die Kistennamen

Reply:
Erledigt. Habe aber die Classnames bei Spezial gelassen, weil ich nicht weiß, ob der Ä Buchstabe Probleme bereiten kann. Könnte auch alles in "praezision", statt "spezial" umbenennen, wenn du das noch wünscht.

Reply:
@shukari Erledigt.